CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * Node: Added LPOS command ([#1927](https://github.com/valkey-io/valkey-glide/pull/1927))
 * Node: Added FUNCTION LOAD command ([#1969](https://github.com/valkey-io/valkey-glide/pull/1969))
 * Node: Added FUNCTION FLUSH command ([#1984](https://github.com/valkey-io/valkey-glide/pull/1984))
+* Node: Fix ZADD bug ([#1995](https://github.com/valkey-io/valkey-glide/pull/1995))
```

Comment:
This should be in category "fixes"

Reply:
Should we be a bit more descriptive or does it not really matter?
```suggestion
* Node: Fix ZADD bug where command could not be called with only the "changed" optional parameter([#1995](https://github.com/valkey-io/valkey-glide/pull/1995))
```